PSix/FacebookEventParser.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FacebookEventParser.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/20/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+
+class FacebookEventsParser {
```

Comment:
The reasoning behind `struct` was to conform to our swift guidelines. `FacebookEventsParser` doesn't really have an "identity" so it should be a struct rather than a class.

As I said before, It's completely up to you.

Now that I think about it, you could even add this as an extension to the `Event` class.